doc/index.rst

```diff
@@ -31,7 +31,17 @@ Currently, PennyLane-Cirq provides two Cirq devices for PennyLane:
     :name: 'cirq.mixedsimulator'
     :description: Cirq's density matrix simulator backend.
     :link: devices/mixed_simulator.html
- 
+
+.. devicegalleryitem::
+    :name: 'cirq.qsim'
+    :description: The qsim simulator backend.
+    :link: devices/qsim.html
+
+.. devicegalleryitem::
+    :name: 'cirq.qsimh'
+    :description: The qsimh simulator backend.
+    :link: devices/qsim.html
```

Comment:
Wasn't sure about having two cards link to the same page. I only added a single page (for both qsim and qsimh), since they work very similarly. Do you think it's better having two separate pages, or perhaps just keeping the qsim card here, even though technically qsimh is a separate device? :thinking:

Reply:
I think that might be the best way to go as well actually. I've updated it now (but kept the line `Currently, PennyLane-Cirq provides four Cirq devices for PennyLane:`, since there are still, technically, four devices).